src/igvfd/schemas/in_vitro_system.json

```diff
@@ -590,6 +590,25 @@
                 "appscript": "DMEM with serum",
                 "igvf_utils": "DMEM with serum"
             }
+        },
+        "biosample_qualifier": {
+            "title": "Biosample qualifier",
+            "description": "An array of various cell state. This property provides additional information at a finer-grained level about a cell compared to what ontologies currently capture. Example exhausted T-cells",
```

Comment:
Just some minor wording suggestions:
* various cell states (plural)
* Move "about a cell" before "at a finer-grained level"
* "For example, exhausted T-cells."

Same comment on the description within "items" on line 601